packages/driver/test/cypress/integration/commands/actions/type_spec.js

```diff
@@ -1646,6 +1646,23 @@ describe('src/cy/commands/actions/type', () => {
         })
       })
 
+      // https://github.com/cypress-io/cypress/issues/5449
+      it('can type into input inside iframe with hover state', () => {
+        cy.$$('<iframe src="/fixtures/dom.html"/>').prependTo(cy.$$('body'))
+
+        // type into aut
+        cy.get('input:first').click()
+
+        // type into iframe
+        cy.get('iframe:first')
+        .should((iframe) => expect(iframe.contents().find('input:first')).to.exist)
+        .then((iframe) => cy.wrap(iframe.contents().find('input:first'), { log: false }))
+        .type('foo')
```

Comment:
I suppose this also tests the `click`, since type will execute a `click`? Just wondering if there needed to be a test case added to cover `click` since that's also what this is fixing.

Reply:
correct, but maybe I should move this test into click_spec and use click instead

Reply:
let me do that